src/js/components/cycleCount/toCountTab/check/CheckStepTable.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+import DataTable from 'components/DataTable/v2/DataTable';
+import useTranslate from 'hooks/useTranslate';
+
+import 'components/cycleCount/cycleCount.scss';
+
+const CheckStepTable = ({
+  product,
+  dateCounted,
+  tableData,
+  columns,
+}) => {
+  const translate = useTranslate();
+  return (
+    <div className="list-page-list-section">
+      <p className="check-step-title pt-4 pl-4">
+        {product?.productCode}
+        {' '}
+        {product?.name}
+      </p>
+      <div className="py-4 pl-4 d-flex align-items-center">
+        <div className="d-flex align-items-center">
+          <span className="check-step-label mr-2">
+            {translate('react.cycleCount.dateCounted.label', 'Date counted')}
+          </span>
+          {' '}
+          <span>{moment(dateCounted).format('DD/MM/YYYY')}</span>
```

Comment:
I thought we had a constant for the date formats. Maybe it would be good to have one. Thoughts?

Reply:
use `formatDate` from `translation-utils`